### ISSUE TITLE
Better copy explanation for empty list in Built-In Elixir Modules

### ISSUE THREAD

**Issue by amir-repository** [DockYard-Academy/curriculum#848]
**Describe the issue**
the copy 
> An empty list technically counts as a keyword list, because it does not have any elements that don't follow the `{:atom, term}` keyword list pattern.

is too hard to understand for new reader, I think there will be a better copy for it

**Reproduction Steps (If Applicable)**
1. Go to Built-In Elixir Modules
2. Scroll down to Keyword exercise
4. the copy explanation is on the exercise that before the last one

**Expected behavior**
No brainer copy explanation

**Screenshots**
![Screenshot from 2023-02-28 05-06-13](https://user-images.githubusercontent.com/37258290/221697662-56733a87-ddba-4600-9e18-30e09f8cffcc.png)

**Comment by BrooklinJazz** [comment]
I agree we can simplify the wording here! I'm open to a PR on this.
